ReactQt/runtime/src/reactuimanager.cpp

```diff
@@ -120,6 +120,20 @@ void ReactUIManager::updateView
   m_bridge->visualParent()->polish();
 }
 
+void ReactUIManager::setChildren
+(
+  int containerReactTag,
+  const QList<int>& childrenTags
+)
+{
+  //This is a simple implementation which fixes a broken example. It's not properly tested and may need revisiting
```

Comment:
@narek-h  It might be good to use comments with TODO keyword to revisit in the future.
For example, `// TODO: This is a simple implementation which fixes a broken example. It's not properly tested and may need revisiting`